tests/testthat/test-add_quantiles.R

```diff
@@ -10,10 +10,98 @@
 
 # Requirement T1 ---------------------------------------------------------------
 
-context("add_quantiles - T1. No errors when confidence intervals are added to the plots.")
+testthat::context("add_quantiles - T1. tbd.")
```

Comment:
Should we make sure all tests are in place before merging?

Reply:
Steven, you're so impatient :D Yeah, I'll pring you when all possible use cases are functioning and test coverage is 100 %